support-frontend/assets/components/subscriptionCheckouts/endSummary/endSummary.jsx

```diff
@@ -0,0 +1,26 @@
+// @flow
+
+import React from 'react';
+import * as styles from './endSummaryStyles';
+
+const Dot = () => <div css={styles.dot} />;
+
+function EndSummary(props: {description: string}) {
+  return (
+    <ul css={styles.list}>
+      <li css={styles.listMain}><Dot />14-day free trial</li>
+      <li css={styles.listMain}>
+        <span><Dot />First 3 months half price</span>
```

Comment:
This should copy come from the promotion not be hard coded

Reply:
True! Thanks Rupert